.github/workflows/trigger-learn-update.yml

```
@@ -0,0 +1,37 @@
+---
+name: Trigger Netdata Learn documentation update
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "**.md"
```

```suggestion
      - "**.mdx?"
```

We want all the documentation files, correct?

---

Ah, yes, I forgot we had some odd cases like that.